### PR TITLE
review modif 3. 5. 7, 8

### PR DESCRIPTION
**Pull request by rtei-pw**
review modif 3. 5. 7, 8

### CODE REVIEW

**Review by Milos-RTEi — APPROVED** [review]

File: src/ui/simulator/windows/inspector/frame.cpp [hunk 1]
```diff
@@ -791,8 +791,7 @@ void Frame::apply(const InspectorData::Ptr& data)
         // Area color
         Accumulator<PAreaColor>::Apply(pPGAreaColor, data->areas);
         // Adequacy patch
-        if (!multiple)
-            Accumulator<PAdequacyPatchMode>::Apply(pPGAreaAdequacyPatchMode, data->areas);
+        Accumulator<PAdequacyPatchMode>::Apply(pPGAreaAdequacyPatchMode, data->areas);
```
Comment (hugo-antoine-rtei) [inline src/ui/simulator/windows/inspector/frame.cpp:794]: It doesn't seem to work when I tested it. When I select multiple areas in the UI and change the AdequacyPatchMode, only one of them get affected. And I can't figure out why.

**Review by hugo-antoine-rtei — APPROVED** [review]
It doesn't seem to work for #`7, but I think that at this point we can ask RTE for more info, unless you can figure out what's going on.

File: src/ui/simulator/windows/inspector/frame.cpp (hunk 1 above)
Comment (rtei-pw) [reply src/ui/simulator/windows/inspector/frame.cpp:794]: > It doesn't seem to work for #`7, but I think that at this point we can ask RTE for more info, unless you can figure out what's going on.

I will wait until further info from RTE

Comment (rtei-pw) [reply src/ui/simulator/windows/inspector/frame.cpp:794]: solved

**Pull request merged**